projects/optic/src/commands/capture/init.ts

```diff
@@ -0,0 +1,71 @@
+import yaml from 'yaml';
+import { updateOpticConfig } from '../../utils/write-optic-config';
+import { logger } from '../../logger';
+
+export function initCaptureConfig(
+  oasFile: string,
```

Comment:
This file name need to be relative to the git root https://github.com/opticdev/optic/blob/main/projects/optic/src/commands/capture/capture.ts#L176